packages/list-group-item/src/ListGroupItem.js

```diff
@@ -8,7 +8,9 @@ const ListGroupItem = ({ borderColor, className, ...props }) => {
 };
```

Comment:
since the reactstrap props are available through the component make sure to add an argstable for that component as well.